packages/sdk-lsk/source/address.service.ts

```diff
@@ -18,6 +19,17 @@ export class AddressService extends Services.AbstractAddressService {
 		return { type: "bip39", address: getLisk32AddressFromPassphrase(mnemonic) };
 	}
 
+	public override async fromMultiSignature({
+		senderPublicKey,
+	}: Services.MultisignatureAddressInput): Promise<Services.AddressDataTransferObject> {
+		assert.ok(senderPublicKey);
+
+		return {
+			type: "lip17",
+			address: senderPublicKey,
```

Comment:
Why does this return a public key instead of an address derived from that public key? This is the address service and return fake values from this is misleading. When you transfer funds from or to MuSig wallets you are still talking to an address, not a public key.

Reply:
Ah, ignore that, uncommitted changes that were never pushed. Pushed now @faustbrian

Reply:
There's still a scenario I need to check based on some notes from months ago. When I initially registered some MuSig wallets it seemed to be possible to have a MuSig wallet that isn't the wallet that started the MuSig wallet. Though I'm not sure if I would want to support that.

Reply:
That would be interesting. But yeah looks like an edge case that unclear from their lip17

Reply:
I'm not even sure if it's intended because I think I was only able to do it directly through code, not through their own wallet.